packages/navy/src/util/nipio.js

```diff
@@ -1,27 +1,27 @@
 /* @flow */
 
-const BASE = 'xip.io'
+const BASE = 'nip.io'
 
 function isValidIpv4Addr(ip) {
   return /^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/g.test(ip)
 }
 
-export async function getXIPSubdomain(externalIP: string) {
+export async function getNIPSubdomain(externalIP: string) {
   if (externalIP === '127.0.0.1') {
     // shorten
-    return `0.${BASE}`
+    return `0.0.0.0.${BASE}`
```

Comment:
Can we remove this special handling of `127.0.0.1` please?
The only reason it was done with `xip.io` was because `foo.0.xip.io` is shorter to type than `foo.127.0.0.1.xip.io`, even though strictly they're not resolving to the same IP address, in most cases the desired effect was the same.